samples/requirement-targeting-oneway-dependency/nodejs/README.md

```diff
@@ -0,0 +1,227 @@
+---
+page_type: sample
+products:
+- office-365
+languages:
+- javascript
+title: Specify relationships between app capabilities
+description: Microsoft Teams RT sample app in Node.js which specify relationships between app capabilities using (elementRelationshipSet) and hostMustSuppot scenarios.
+extensions:
+  contentType: samples
+  createdDate: 08/21/2024 10:02:21 PM
+urlFragment: officedev-microsoft-teams-samples-requirement-targeting-oneway-dependency
+---
+
+# Microsoft Teams RT Sample App Feature.
+
+- Microsoft Teams RT sample app in Node.js which specify relationships between app capabilities using (elementRelationshipSet) and hostMustSuppot scenarios.
+
+## Included Features
+* Tabs
+* Bots
+* Compose Extensions
+* Element RelationShip Property
+    * OneWay Dependency
+    * Host Must Support Functionalities with dialogUrl
+
+## Interaction with app
+
+![RTSampleApp](Images/RT-Sample-App.gif)
+
+## Prerequisites
+
+-  Microsoft Teams is installed and you have an account (not a guest account)
+
+-  To test locally, [NodeJS](https://nodejs.org/en/download/) must be installed on your development machine (version 16.14.2  or higher)
+
+-  [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunneling solution
+
+-  [M365 developer account](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to a Teams account with the appropriate permissions to install an app.
+
+- [Teams Toolkit for VS Code](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) or [TeamsFx CLI](https://learn.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli?pivots=version-one)
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
```

Comment:
Shall we also add an instruction to inform developers to move each `manifest.json` to the `appPackage` folder in order to see each manifest scenario separately? This is just so it's more clear to developers of why there's this `Manifests` folder and how to use it.

Reply:
Moved to **appPackage** folder